redis/internal/redisprom/pools.go

```diff
@@ -3,12 +3,15 @@ package redisprom
 import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	//lint:ignore SA1019 This library is internal only, not actually deprecated
```

Comment:
@kylelemons I think we should move `GlobalRegistry` to an actual internal package from `internalv2compat`, as it's not needed outside of this repo (unlike other things in `internalv2compat`), so that we don't have to do this every time it's used.

`internal/prometheusbp` is a good place for it.